gladier/client.py

```diff
@@ -258,7 +258,46 @@ def sync_flow(self):
         self.flows_manager.flow_definition = self.get_flow_definition()
         self.flows_manager.sync_flow()
 
-    def run_flow(self, flow_input=None, use_defaults=True, **flow_kwargs):
+    def run_flow(self, flow_input=None, use_defaults=True,
+                 run_managers: Optional[typingIterable[str]] = None,
+                 run_monitors: Optional[typingIterable[str]] = None,
+                 dry_run: bool = False,
+                 label: Optional[str] = None,
+                 tags: Optional[List[str]] = None,
+                 **flow_kwargs):
```

Comment:
I'm a bit split on some of the changes here. I really like the type checking and the explicit listing of signature arguments. However, it does mix the Globus Automate Client arguments with the Gladier option `use_defaults`.

I think ideally we would document the Gladier specific arguments then punt other args to the GAC documentation, but maybe it's just easier for everyone if we document everything in the same place? What do you think @jimPruyne?

Reply:
I guess I'm similarly torn. I would definitely keep the arguments with the types defined (though I'd probably use `from __future__ import annotations` and use annotations like `list[str]` rather than `List[str]`.

And, yeah, I'd probably insert something in the docstrings referring to the GAC doc strings, and not repeat that here.

So, it seems we're thinking along the same lines.

Reply:
The changes merged earlier did just that and referenced the GAC docs for flows-specific values. So maybe we keep the types and leave the other documentation as is?